numpyro/distributions/continuous.py

```diff
@@ -668,6 +669,68 @@ def cdf(self, x):
         return 1 - self.base_dist.cdf(1 / x)
 
 
+class Gompertz(Distribution):
+    r"""Gompertz Distribution sampling utilities.
+
+    This implementation is based off the derivations from Wikipedia page for the
+    Gompertz distribution. See https://en.wikipedia.org/wiki/Gompertz_distribution.
+
+    However, we call the parameter "eta" a concentration parameter.
+    We also let the user specify a rate parameter instead of the scale parameter "b".
+    The two are related by rate = 1 / scale. By default, `rate` = 1.
```

Comment:
this seems to be consistent with wikipedia but wikipedia doesn't make a whole lot of sense here since usually the `scale` parameter should have units of `x`. compare to e.g. usage of 'scale' in https://en.wikipedia.org/wiki/Exponential_distribution

Reply:
sounds good